src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
I don't think by itself it useful yeah. But I'd just like to be flexible in case we add http://underscorejs.org/#functions in the future.